src/app/p-common/menubar/menubar.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { MenuItem } from 'primeng/api';
+
+@Component({
+  selector: 'app-menubar',
+  templateUrl: './menubar.component.html',
+  styleUrls: [ './menubar.component.css' ]
+})
+export class MenubarComponent implements OnInit {
+
+  items: MenuItem[];
+
+  constructor() {
+  }
+
+  ngOnInit() {
+    this.items = [
+      {
+        label: 'Projekty',
+        icon: 'pi pi-fw pi-list',
+        routerLink: [ '/projects' ]
+      },
+      {
+        label: 'Dostępność',
+        icon: 'pi pi-fw pi-user',
+        routerLink: [ '/availability' ]
+      },
+      {
+        label: 'Excel',
+        icon: 'pi pi-fw pi-table',
+        routerLink: [ '/excel' ]
+      },
+      { separator: true },
+      {
+        label: 'Nowy sprint',
+        icon: 'pi pi-fw pi-plus',
+        routerLink: [ '/new-sprint' ]
```

Comment:
chyba nie ma możliwości sprawdzenia, czy ktoś ma uprawnienia do takiej akcji
może na razie może umówmy się tak, że to zostanie, a jakieś takie szlify (jak właśnie ukrycie tego przycisku) zostawimy na koniec, już po połączeniu wszystkich komponentów, ok?

Reply:
Ok, jak najbardziej.